shared/templates/grub2_bootloader_argument/oval.template

```diff
@@ -3,63 +3,63 @@
     {{{ oval_metadata("Ensure " + ARG_NAME_VALUE + " is configured in the kernel line in /etc/default/grub.") }}}
     <criteria operator="AND">
       {{% if product in ["rhel7", "ol7"] or 'ubuntu' in product %}}
-        <criterion test_ref="test_grub2_{{{ SANITIZED_ARG_NAME }}}_argument_grub_cfg"
-        comment="Check if {{{ ARG_NAME_VALUE }}} is present in the boot parameters in the {{{ grub2_boot_path }}}/grub.cfg for all kernels" />
+        <criterion test_ref="test_grub2_{{{ SANITIZED_ARG_NAME + ID_SUFFIX }}}_argument_grub_cfg"
```

Comment:
This `SANITIZED_ARG_NAME + ID_SUFFIX` is very ugly, just use jinja's `{{% set unique_oval_id_stem=SANITIZED_ARG_NAME + ID_SUFFIX %}} (or something like that) -  that will be much better